### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by prb112**
Signed-off-by: Paul Bastide <pbastide@us.ibm.com>

### CODE REVIEW

**Review by lmsurpre — APPROVED** [review]
LGTM

**Comment by JohnTimm** [comment]
I would remove "fully-defined" from the description.

File: README.md [hunk 1]
```diff
@@ -58,6 +58,7 @@ The IBM FHIR Server is modular and extensible. The following tables provide an o
 |fhir-model|An object model generated from the FHIR R4 specification and corresponding parsers and generators for XML and JSON|true|
 |fhir-registry|A resource registry, registry provider interfaces, and pre-registered resources shipped with the FHIR specification|false|
 |fhir-term|A terminology service provider interface with a default implementation that implements terminology services from fully-defined CodeSystems in the registry|false|
+|fhir-term-graph|A terminology service provider that implements terminology services from fully-defined CodeSystems using JanusGraph|false|
```
Comment (prb112) [inline README.md:61]:
```suggestion
|fhir-term-graph|A terminology service provider that implements terminology services from CodeSystems using JanusGraph|false|
```

Comment (prb112) [inline README.md]:
```suggestion
|fhir-term-graph|A terminology service provider that implements terminology services using JanusGraph|false|
```

File: README.md [hunk 2]
```diff
@@ -58,6 +58,7 @@ The IBM FHIR Server is modular and extensible. The following tables provide an o
 |fhir-model|An object model generated from the FHIR R4 specification and corresponding parsers and generators for XML and JSON|true|
 |fhir-registry|A resource registry, registry provider interfaces, and pre-registered resources shipped with the FHIR specification|false|
 |fhir-term|A terminology service provider interface with a default implementation that implements terminology services from fully-defined CodeSystems in the registry|false|
+|fhir-term-graph|A terminology service provider that implements terminology services from CodeSystems using JanusGraph|false|
```
Comment (prb112) [inline README.md:61]:
```suggestion
|fhir-term-graph|A terminology service provider that implements terminology services using JanusGraph|false|
```

**Pull request merged**